crates/client/sync/src/fetch/fetchers.rs

```diff
@@ -40,15 +41,25 @@ pub struct FetchConfig {
     pub sync_polling_interval: Option<Duration>,
     /// Number of blocks to sync (for testing purposes).
     pub n_blocks_to_sync: Option<u64>,
+    /// Number of blocks between db flushes
+    pub flush_every_n_blocks: u16,
```

Comment:
why u16? block numbers are u64 and so are the spans between blocks
doesnt this add a lot of unnecessary conversions?

Reply:
I kind of wanted to restrict the max interval a user could set but I think there is a better way to do this by setting this to a `u64` and updating the clap parser which is used to retrieve this value.

Reply:
This has been fixed. Set the value to `u64` and the legal value as 10 000